src/newMockStore.ts

```diff
@@ -0,0 +1,21 @@
+import { ActionContext, Commit, Dispatch } from 'vuex';
+
+export default function newMockStore(
+	{ commit, dispatch, state, getters, rootState, rootGetters }: {
+		commit?: Commit;
+		dispatch?: Dispatch;
+		state?: any;
+		getters?: object;
```

Comment:
What happened to the `GetterTree` and why?

Reply:
Interestingly ActionContext interface only requires any not specifically a GetterTree

Were you thinking we should have instead met the StoreOptions interface?

Reply:
I was (almost) not thinking at all but comparing with [`newMockStore` in termbox](https://gerrit.wikimedia.org/r/plugins/gitiles/wikibase/termbox/+/bd69d5a25d8bdd10d8de5ac397c4b5b8b4c3bc33/tests/unit/store/newMockStore.ts#8). Maybe `ActionContext` is sloppy about this (github karma right there) but there is no doubt in my mind that `getters` and `rootGetters` contain a tree of getters...

Reply:
I didn't verify this but the reasoning here makes sense to me: https://gerrit.wikimedia.org/r/c/mediawiki/extensions/Wikibase/+/534603

That would also explain why https://github.com/vuejs/vuex/blob/0e109e2a38dafdc0c2bd6bd3892bc66cfe252b16/types/index.d.ts#L55 is not hinting `GetterTree`